contrib/generic_proxy/filters/network/source/route.h

```diff
@@ -94,6 +97,21 @@ class NullRouteMatcherImpl : public RouteMatcher {
   RouteEntryConstSharedPtr routeEntry(const Request&) const override { return nullptr; }
 };
 
+class VirtualHostImpl : Logger::Loggable<Envoy::Logger::Id::filter> {
+public:
+  VirtualHostImpl(const ProtoVirtualHost& virtual_host_config,
+                  Envoy::Server::Configuration::ServerFactoryContext& context,
+                  bool validate_clusters_default = false);
+
+  RouteEntryConstSharedPtr routeEntry(const Request& request) const;
+  absl::string_view name() { return name_; }
```

Comment:
nit, const?

Reply:
`string_view` needn't additional `const` but it already have const semantics. Hah, I see, `const` for the method, good point, thanks.